templates/tempest.j2

```diff
@@ -10,8 +10,11 @@ flavor_ref_alt = {{ tempest_second_flavor_uuid }}
 fixed_network_name = {{ tempest_network_name }}
 
 [object-storage]
-swift_operator_role = member
-swift_reseller_admin_role = admin
+swift_operator_role = {{ rally_tempest_swift_operator_role }}
+swift_reseller_admin_role = {{ rally_tempest_swift_reseller_admin_role }}
+
+[object-storage-feature-enabled]
```

Comment:
Could the "[object-storage-feature-enabled]" section of the template be put inside an if statement?  Or what happens if this is enabled and there's no object storage service available?

Reply:
Discoverability is on by default, its part of the OpenStack Swift Protocol/API
Ceph Rados GW doesn't support it yet in the Jewel release